### ISSUE TITLE
Split long lines in inventory

### ISSUE THREAD

**Issue by rfleschenberg** [ansible/ansible#9895]
There does not seem to be a way to split long lines in inventory files:

```
[webservers]
some_host_with_many_vars var1=foo var2=bar
    var3=baz
```

This should be possible according to the `ConfigParser` docs, but it does not work in Ansible.

**Comment by bcoca** [comment]
# List Information

Hi!

Thanks very much for your interest in Ansible.  It sincerely means a lot to us. 

This appears to be a user question, and we'd like to direct these kinds of things to either the mailing list or the IRC channel.
- IRC: #ansible on irc.freenode.net  
- mailing list: https://groups.google.com/forum/#!forum/ansible-project

If you can stop by there, we'd appreciate it.  This allows us to keep the issue tracker for bugs, pull requests, RFEs and the like.

As for your question, inventory files do NOT use config parser and have no current way to allow for multiline content, we advise using group_vars/<groupname>  or host_vars/<hostname> if you have many group or host specific variables.

Thank you once again and we look forward to seeing you on the list or IRC.  Thanks!

**Comment by rfleschenberg** [comment]
Thanks. I had actually discussed this beforehand on IRC, where it was suggested that I open an issue.

I dislike having to spread my inventory hosts and their variables over multiple places. I know that other people dislike `host_vars` for the same reason.

Is there a technical reason for not using `ConfigParser`? I guess I could use the dynamic inventory mechanism to feed ansible from an INI file that I parse with ConfigParser, but it would be nice if I did not have to.
